test-data/unit/check-expressions.test

```diff
@@ -1157,6 +1157,25 @@ b'%c' % (123)
 [case testUnicodeInterpolation_python2]
 u'%s' % (u'abc',)
 
+
+-- F-String
+-- --------
+
+
+[case testFStringParseOk]
+# flags: --fast-parser
+f'foobar'
+
+[case testFStringTypecheckExpression]
+# flags: --fast-parser
+# flags: --python-version 3.6
+a: str
+a = 'foo'
+b: str
+b = f'{a}bar'
```

Comment:
I'd also try something with a type error inside the `{}`, e.g.
```python
a = 1
b = ''
f'{a + b}'
```
Note that plain `a + b` gives an error, so `f'{a + b}'` should also gives an error.